conans/client/tools/oss.py

```diff
@@ -24,12 +25,32 @@ def args_to_string(args):
 
 def cpu_count(output=None):
     output = default_output(output, 'conans.client.tools.oss.cpu_count')
+
+    def is_running_in_docker():
+        cgroup = os.path.join(os.path.sep, "proc", "self", "cgroup")
+        return (os.path.exists('/.dockerenv') or
+                (os.path.isfile(cgroup) and any('docker' in line for line in open(cgroup))))
```

Comment:
Is the opening of this file save? Should this not be surrounded in a try/catch?

Reply:
done